src/hyperdrive-cli/commands/stakewise/validator/get-signed-exit-messages.go

```diff
@@ -32,12 +32,15 @@ func getSignedExitMessages(c *cli.Context) error {
 	// Get the client
 	sw := client.NewStakewiseClientFromCtx(c)
 
-	activeValidatorResponse, err := sw.Api.Status.GetActiveValidators()
+	activeValidatorResponse, err := sw.Api.Status.GetValidatorStatuses()
 	if err != nil {
 		return fmt.Errorf("error while getting active validators: %w", err)
 	}
+	var activeValidators []beacon.ValidatorPubkey
+	activeValidators = append(activeValidators, activeValidatorResponse.Data.ActiveOngoing...)
+	activeValidators = append(activeValidators, activeValidatorResponse.Data.ActiveExited...)
```

Comment:
Not sure I understand here - why are you allowing people to select validators that have been slashed or already exited when generating presigned exit messages? I can understand something like pending or queued, but already exited doesn't seem right.